.github/workflows/brakeman.yml

```diff
@@ -22,10 +22,31 @@ jobs:
 
       - name: Run Brakeman
         continue-on-error: true
-        run: bundle exec brakeman . --except CheckRenderInline --quiet -f sarif >> brakeman.sarif
+        run: |
+          if [ "${{ github.repository_visibility }}" == 'public' ]; then
```

Comment:
While this particular example isn't dangerous (I think, assuming there isn't any way for an outside user to influence the value of `github.repository_visibility`), it's best to avoid substituting values directly into shell commands in GitHub Actions like this because it's all too easy to create command-injection vulnerabilities that way.

Reply:
https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#example-of-a-script-injection-attack